pkg/asset/agent/image/ignition.go

```diff
@@ -108,10 +117,38 @@ func (a *Ignition) Generate(dependencies asset.Parents) error {
 		return err
 	}
 
+	agentMirror := &mirror.AgentMirror{}
+	dependencies.Get(agentMirror)
+
+	// Mount the mirror registries file for assisted-service
+	mirrorRegistriesMount := ""
+	if _, err := os.Stat(mirror.RegistriesConfFilename); err == nil {
```

Comment:
Since the mirror dependency it's a writable asset, wouldn't be better to rely just on some asset property (like the `File` or `FileList`, or another one created ad-hoc) to decide if some content it's available or not, and avoiding to access directly the disk from here?

Reply:
Yep, good point, thanks. Fixed.